### DISCUSSION TITLE
Multiple Windows with different compononents

### DISCUSSION THREAD

**Discussion by AlessioBottin** [cawa-93/vite-electron-builder#836] [Q&A] [answered]
Hi, I was wondering if there is a way to renderer only a Vue component in a window;
I thought I could try with vue router by passing the url and the route i want to display in win.loadURL(), but would it work in prodution?
Is there a better solution?
Is creating another vue app in renderer and compiling it so that each window is a different app a solution?

**Comment by cawa-93** [comment, marked as answer]
I using vue-router in my apps. Example: https://github.com/cawa-93/anime-library/tree/main/packages%2Frenderer

**Reply by AlessioBottin** [reply]
Thank you so much!
